### PR TITLE
74378  -aio webjob logging and config

### PR DESCRIPTION
**Pull request by abhijeet14992**
- Added configuration related to Azure Event Hub, Application Insights,  KeyVault and Serilog.

### CODE REVIEW

**Review by shailen14599 — CHANGES_REQUESTED** [review]
Few minor changes suggested

File: UKHO.PeriodicOutputService/UKHO.AdmiraltyInformationOverlay.Fulfilment/AioFulfilmentJob.cs [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        }
+
+        public void ProcessFulfilmentJob()
+        {
+            try
+            {
+                _logger.LogInformation(EventIds.AIOFulfilmentJobStarted.ToEventId(), "AIO webjob started | _X-Correlation-ID : {CorrelationId}", CommonHelper.CorrelationID);
+
+                _logger.LogInformation(EventIds.AIOFulfilmentJobEnded.ToEventId(), "AIO  webjob ended | _X-Correlation-ID : {CorrelationId}", CommonHelper.CorrelationID);
```
[the remark this answers is not in the document]
Comment (abhijeet14992) [reply UKHO.PeriodicOutputService/UKHO.AdmiraltyInformationOverlay.Fulfilment/AioFulfilmentJob.cs]: Resolved

File: UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Logging/EventIds.cs [hunk 2]
```diff
@@ -300,7 +300,16 @@ public enum EventIds
...
-        CancelledProductsFound = 809073
+        CancelledProductsFound = 809073,
+        /// <summary>
+        /// 809074 - AIO fulfilment job started
+        /// </summary>
+        AIOFulfilmentJobStarted = 809074,
+        /// <summary>
+        /// 809075 - AIO fulfilment job ended
+        /// </summary>
+        AIOFulfilmentJobEnded = 809075,
```
[the remark this answers is not in the document]
Comment (abhijeet14992) [reply UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Logging/EventIds.cs]: Resolved

**Review by shailen14599 — APPROVED** [review]
LGTM

**Review by DanOwenUKHO — APPROVED** [review]
LGTM

**Pull request merged**